release_templates/mxnet_release_images.yml

```diff
@@ -0,0 +1,34 @@
+---
+release_images:
+  1:
+    framework: "mxnet"
+    version: "1.6.0"
+    training:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py27", "py36"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu101"
+      example: False
+      disable_sm_tag: False # Set to True to prevent SageMaker Abbreviated Tags from being attached
+                            # to images being published.
+      force_release: False  # Set to True to force images to be published even if the same image
```

Comment:
Update the comment here to say that: the new release will have a bumped up minor version.

Reply:
Thanks. Will add this.